ovos_tts_plugin_server/__init__.py

```diff
@@ -28,55 +38,76 @@ def v2(self) -> bool:
 
     @property
     def verify_ssl(self) -> bool:
+        """Whether or not to verify SSL certificates when connecting to the server. Defaults to True."""
         return self.config.get("verify_ssl", True)
 
-    def get_tts(self, sentence, wav_file, lang=None, voice=None):
-        lang = lang or self.lang
-        voice = voice or self.voice
-        params = {"lang": lang, "voice": voice}
+    def get_tts(
+        self,
+        sentence,
+        wav_file,
+        lang: Optional[str] = None,
+        voice: Optional[str] = None,
+    ) -> Tuple[Any, None]:
+        """Fetch TTS audio using OVOS TTS server.
+        Language and voice can be overridden, otherwise defaults to config."""
+        params: Dict[str, Optional[str]] = {
+            "lang": lang or self.lang,
+            "voice": voice or self.voice,
+        }
         if not voice or voice == "default":
             params.pop("voice")
         if self.host:
             if isinstance(self.host, str):
-                servers = [self.host]
+                servers: List[str] = [self.host]
             else:
                 servers = self.host
         else:
-            servers = self.public_servers 
-        data = self._get_from_servers(params, sentence, servers)
-        with open(wav_file, "wb") as f:
-            f.write(data)
+            servers = self.public_servers
+        data: bytes = self._fetch_audio_data(params, sentence, servers)
+        self._write_audio_file(wav_file, data)
         return wav_file, None
 
-    def _get_from_servers(self, params: dict, sentence: str, servers: list):
+    def _write_audio_file(self, wav_file: str, data: bytes) -> None:
+        with open(file=wav_file, mode="wb") as f:
+            f.write(data)
+
+    def _fetch_audio_data(self, params: dict, sentence: str, servers: list) -> bytes:
+        """Get audio bytes from servers."""
         random.shuffle(servers)  # Spread the load among all public servers
```

Comment:
It was mentioned with the STT server that only the public servers should be shuffled.  Does this shuffle the list no matter where it came from?

Reply:
Yes, it does. That's the original behavior, I didn't change anything